maintainers/maintainer-list.nix

```diff
@@ -4915,6 +4915,13 @@
     githubId = 37017396;
     name = "gbtb";
   };
+  gdamjan = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.